src/genie/libs/parser/iosxe/tests/ShowCtsRoleBasedPermissions/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,97 @@
+expected_output = {
```

Comment:
please ignore. this is under review in https://github.com/CiscoTestAutomation/genieparser/pull/219